### PR TITLE
Log fixes

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by joaozin23** [comment]
esboco 12

**Comment by joaozin23** [comment]
rpcs3-v0.0.4-2017-11-20-969e38b7_win64

**Comment by joaozin23** [comment]
https://ci.appveyor.com/project/rpcs3/rpcs3/build/artifacts

**Comment by hcorion** [comment]
It looks like you didn't extract rpcs3 before running it @joaozin23 ?

**Comment by joaozin23** [comment]
extract it correctly but zip it this problems

**Comment by Megamouse** [comment]
crash on startup is fixed.
log no longer shows 130mb (yay)

**Comment by joaozin23** [comment]
esboco1

File: rpcs3/main.cpp [hunk 1]
```diff
@@ -18,6 +19,40 @@ namespace logs
...
+	{
+		s_init.wait();
+		static int argc = 1;
+		static char arg1[] = {"ERROR"};
+		static char* argv[] = {arg1};
+		static QApplication app0{argc, argv};
+	}
+
+	QMessageBox msg;
+	msg.setWindowTitle("RPCS3: Fatal Error");
```
Comment (Megamouse) [inline rpcs3/main.cpp]: maybe add tr() here
and down at setText(tr(...))

same could be done in experimental build warning and SSSE3 warning.

File: rpcs3/rpcs3qt/main_window.cpp [hunk 2]
```diff
@@ -419,39 +373,29 @@ void main_window::InstallPkg(const QString& dropPath)
...
 		});
+
 		// Wait for the completion
 		while (std::this_thread::sleep_for(5ms), std::abs(progress) < 1.)
 		{
 			if (pdlg.wasCanceled())
 			{
 				progress -= 1.;
-#ifdef _WIN32
-				taskbar_progress->hide();
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/main_window.cpp]: these changes cause the taskbarbutton to remain filled/active after installation

Comment (Megamouse) [reply rpcs3/rpcs3qt/main_window.cpp]: image

**Comment by AllenWillLiu** [comment]
I've tested many games, and it's too much to reply, well, here, I'll just tell about two games: BLJS10324-Kamen_Rider_Battride_War_Sousei_Memorial_TV_Sound_Edition(I'm gonna call this"KRBWS") and BLJS10335-Super_Robot_Taisen_OG_The_Moon_Dwellers(I'll call this"SRWOG-MD", for the KRBWS, well, it was all fine, very smooth to play, however, it used to get silence when you play for a while, that's all for this game. Now, let's say SRWOG-MD, when I used Renderer: Vulcan，it always stuck in the title screen, well, sometimes I can get through the title screen luckily, but the background pics are really terrible, it's all Greeeen and Mosaics, and I found if I use DX12 to run it, I could get into title stably, and the pics were much more better than Vulcan. But, as you know, DX12 is not really stable for playing, I've heard that Nvidia graphics boards will cause broken pics in SRW2OG and SRWOG-MD, unfortunely, I'm using Nvidia GTX 1060 now, and my Cpu is i7 7700, I believe it could run rpcs3 well enough, it's fun to try about this emulators, and I belive you guys can make everything goes good, have a good night.

**Comment by Asinin3** [comment]
This Pull Request broke something with the logs, they now write traces to the log file. I ran Magic The Gathering 2013 for 2minutes and had 8,000,000 lines of traces written to my log file.

**Comment by Megamouse** [comment]
maybe the log just was so terribly slow before that you closed rpcs3 before it came to that size?

**Comment by Asinin3** [comment]
Maybe? but it's still a huge problem for me especially since the filesize doesn't cap anymore if i play some games for more than 20-30seconds the log will be too big for the upload limit (4mb) on the forum. I don't know what the cause is but it's making it very hard to do compatibility reports and for debugging as well.

**Comment by clienthax** [comment]
7z should get around the file size issue

**Comment by Asinin3** [comment]
It doesn't, 7z LZMA ultra results in a ~150mb 7z file after running magic the gathering 2013 for 1.5min.

**Comment by Nekotekina** [comment]
Need to truncate uploaded files then.

**Pull request merged**